app/lib/pocTransform.js

```diff
@@ -75,21 +84,24 @@ const durationToDiscreet = durationArr => {
 const MINUTE = 60 * 1000;
 
 const discreetMergeCondition = (curDiscreet, curDuration) => {
+  if (curDiscreet.caseId !== curDuration.caseId) {
+    return false;
+  }
   if (curDiscreet.latitude !== curDuration.latitude) {
     return false;
   }
   if (curDiscreet.longitude !== curDuration.longitude) {
     return false;
   }
-  if (curDiscreet.time > curDuration.time + curDuration.duration * MINUTE) {
+  if (Date.parse(curDiscreet.time) > Date.parse(curDuration.time) + curDuration.duration * MINUTE) {
```

Comment:
We're no longer doing translation on points incoming from SafePaths. The `time` attributes here will be represented as date strings and need to be parsed to epoch time to do these calculations.